### PR TITLE
KTOR-3937 Support Snap publication

### PR DESCRIPTION
**Pull request by Ololoshechkin**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -85,4 +85,8 @@ kotlin {
             module.dependsOn(nativeTest)
         }
     }
-}
\ No newline at end of file
+}
+
+// Is required for snapcraft build because it runs "jar" task by default, and it is impossible to avoid it.
+// Note: Snapcraft is a tool for publication binaries for Linux via Snap package manager.
```
Comment (e5l) [inline build.gradle.kts:91]: Please add a link to snapcraft or drop a note  :)

File: snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: ktor # you probably want to 'snapcraft register <name>'
+summary: Creates porjects with ktor framework setup.
+version: '0.0.1' # just for humans, typically '1.2+git' or '1.3.2'
```
Comment (e5l) [inline snapcraft.yaml]: should we align the version with Ktor and use 2.1.0-eap-1?

**Review by e5l — APPROVED** [review]
LGTM, please check minor notes

File: snapcraft.yaml (hunk 2 above)
Comment (Ololoshechkin) [reply snapcraft.yaml]: Will do, but for now this is a very first temporary release which I'm planning to remove later once the publication is set up properly

**Pull request merged**